data/os-signals.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @title Handling OS Signals
+ * @difficulty beginner
+ * @tags cli
+ * @run <url>
+ * @resource {https://doc.deno.land/deno/stable/~/Deno.addSignalListener} Doc: Deno.addSignalListener
```

Comment:
We now promote https://deno.land/api as API doc site instead of https://doc.deno.land/

So please link to https://deno.land/api?s=Deno.addSignalListener instead